packages/docs/src/locales/en/en.json

```diff
@@ -3065,6 +3067,25 @@
       }
     }
   },
+  "tailwind": {
```

Comment:
We need to make docs a bit cleaner, it's very hard to understand right now.

What I would suggest:

Have 2 installation flows explained:
1) You install VuesticUI on top of Tailwind
2) You install Tailwind on top of VuesticUI

Then separately talk about syncing.

Right now it's not clear what steps have to be taken (is tailwind dependency included in plugin, is vuestic included in plugin etc).

Reply:
Docs updates.